internal/logstore/observability/lokistack/lokistack.go

```diff
@@ -0,0 +1,298 @@
+package lokistack
```

Comment:
Do these need to go in the test package? migration package?  I'm not following why we are reconciling anything for any logstore

Reply:
I've removed this and moved those specific functions to the migration package for lokistack